app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Login</h1>
+   <%= form_tag '/login' do %>
+   <%= label_tag :username%>
+   <%= text_field_tag :username %>
```

Comment:
@idprietop when you look at **text_field_tag** once the template is rendered, what will you see?

Reply:
Once the template is rendered the text_fiel_tag creates a standard text field and is used to enter smaller pieces of text.